### PR TITLE
ISSUE 134: Generate enum reverse lookup

### PR DESCRIPTION
**Pull request by cheleb**

### CODE REVIEW

**Comment by keynmol** [comment]
Thanks for this @cheleb!

Unfortunately in my short absence the build has degraded somehow, so once I fix it in #139 (windows left), I will come back to your PR.

File: modules/tests/src/test/scalanative/TestEnums.scala [hunk 1]
```diff
@@ -48,6 +48,13 @@ class TestEnums:
     assertEquals(9, EnumKeywords.`var`.int)
   end test_reserved_keywords
 
+  // @Test def test_getname(): Unit =
```
Comment (keynmol) [inline modules/tests/src/test/scalanative/TestEnums.scala:51]: Are these intentionally commented out?

Comment (cheleb) [reply modules/tests/src/test/scalanative/TestEnums.scala]: Nope, was a mistake, sorry about this.

Comment (keynmol) [reply modules/tests/src/test/scalanative/TestEnums.scala]: No worries at all!

**Comment by keynmol** [comment]
@cheleb I believe I have fixed the build in #139 - would you mind please updating your branch with latest main so we can see the tests run proper?

**Review by keynmol — APPROVED** [review]
Beautiful! thank you so much

**Pull request merged**